napari/_tests/test_conftest_fixtures.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from qtpy.QtCore import QMutex, QThread, QTimer
+
+
+class _TestThread(QThread):
+    def __init__(self):
+        super().__init__()
+        self.mutex = QMutex()
+
+    def run(self):
+        self.mutex.lock()
+
+
+@pytest.mark.disable_qthread_start
+def test_disable_qthread(qapp):
+    t = _TestThread()
+    t.mutex.lock()
+    t.start()
+    assert not t.isRunning()
+    t.mutex.unlock()
+
+
+def test_qthread_running(qtbot):
+    t = _TestThread()
+    t.mutex.lock()
+    t.start()
+    assert t.isRunning()
+    t.mutex.unlock()
+    qtbot.waitUntil(t.isFinished, timeout=2000)
+
+
+@pytest.mark.disable_qtimer_start
+def test_disable_qtimer(qtbot):
+    t = QTimer()
+    t.setInterval(100)
+    t.start()
+    assert not t.isActive()
+
+    th = _TestThread()
```

Comment:
I assume this is testing that threads still run when timers are disabled? If so, I'd suggest either adding an implementation comment to the code to explain this, or alternatively defining a separate test for this.

Reply:
I have added a comment explaining the second part of the test. Did you think that it should be splitted?

Reply:
The comment is great, so no need to split.